modules/ROOT/pages/about-components.adoc

```diff
@@ -1,57 +1,176 @@
+* xref:logger-component-reference.adoc[Logger]: For logging important information about
+your Mule app, such as error messages and status notifications.
+* xref:parse-template-reference.adoc[Parse Template]: For processing a template
+and obtaining a result.
+* xref:transform-component-about.adoc[Transform Message]: For converting input
+data to a new output structure or format.
 
-A Scope (blue color in Design Center) groups a sequence of event processors together to apply some programming behavior to that isolated sequence of event processors.
+== Endpoints
 
-For example, a Try scope lets you isolate and handle any errors that occur in a particular sequence of flows. So, you might wrap an outbound HTTP Request connector (and perhaps some other surrounding components before and after the HTTP Request connector, such as a Transform Message component or a Logging component) in a Try scope, so if an error results, you can apply logging or compensation logic specific to that grouping of components.
+An endpoint is a Core component that initiates (or triggers) processing in a
+Mule flow. The xref:scheduler-concept.adoc[Scheduler] serves as an endpoint. It triggers a flow to start at a configurable interval.
 
-A For Each scope takes a collection of data, usually extracted from the current input event, and applies the same sequence of event processors to every item in the collection. For example, a For Each scope might be used to process each individual row returned from a database query or each individual line from a CSV file.
+Note that some connectors provide listeners that serve as endpoints. Listeners
+can trigger a flow when they receive an external request. For example, an HTTP
+Listener that is configured for a given URL can trigger a flow when someone or \
+some process goes to that URL.
+
+== Error Handling
+
+Error handling components route and process Mule errors that occur in a Mule app:
+
+* Error Handler
+* On Error Continue
+* On Error Propagate
+
+See xref:error-handling.adoc[Error Handlers]
+
+== Flow Control (Routers)
+
+A Flow Control takes the input event and routes it to one or more separate sequences of components. Flow Control components (or Routers) include:
+
+* Choice
+* First Successful
+* Round Robin
+* Scatter-Gather
+
+For example, a xref:choice-router-concept.adoc[Choice router]) applies
+DataWeave logic to pick one of several routes, where each route is a separate sequence of event processors.
+
+A xref:scatter-gather-concept.adoc[Scatter-Gather router] router copies the
+input event to every route, and each route independently processes the event.
+After every route has processed the event, the results from all the routes are
+combined together into one output event.
+
+== Scopes
+
+A Scope is a type of component that groups together a sequence of event processors (modules, components, or connectors) to apply some programming behavior to that isolated sequence of event processors.
```

Comment:
A module is a grouping of components with a specific purpose (like the Java module which regards all Java interoperation). A connector is a module whose purpose involves connecting to external services (whether it's a generic protocol like HTTP or a specific service like Salesforce). What you use in a flow are the components they provide. We really must move away from referring to the flow pieces as "modules" or "connectors" and instead talk about components, operations, sources, and so on.

Reply:
Oh, right. I'll modify that.

Reply:
Done. Changed "event processors (modules, components, or connectors)" to "event processors (including other Core components and operations from modules and connectors)" since the UI refers to the module and connector components as "operations".